Cargo.toml

```diff
@@ -17,21 +17,19 @@ base64-serde = "~0.3.1"
 bytes = "~0.4.12"
 serde = "~1.0"
 serde_json = "~1.0"
-publicsuffix = "=1.5.2" # exact version needed to match url used by reqwest 0.9
+publicsuffix = "^1.5.4" # exact version needed to match url used by reqwest 0.9
```

Comment:
Remove comment about exact version? Should we use `~1.5.4` here?

Reply:
I think we do want that... Anything inside 1.x should be fine I would think?